### PR TITLE
garden: remaining diff from urbit/urbit

### PR DESCRIPTION
**Pull request by tomholford**

### CODE REVIEW

File: desk/desk.docket-0 [hunk 1]
```diff
@@ -1,10 +1,10 @@
 :~  title+'System'
```
Comment (arthyn) [inline desk/desk.docket-0:1]: revert these, this old docket is correct

**Review by arthyn — CHANGES_REQUESTED** [review]
just revert the docket and then gtg

**Comment by tomholford** [comment]
@arthyn Restored the docket glob + version, should be good to go

**Review by arthyn — APPROVED** [review]
lgtm!

**Pull request merged**